### PR TITLE
fix calling toLowerCase() on undefined

### PR DESCRIPTION
**Pull request by Ciki**

### CODE REVIEW

File: nette.ajax.js [hunk 1]
```diff
@@ -384,7 +384,7 @@ $.nette.ext('forms', {
 		}
 		
 		// https://developer.mozilla.org/en-US/docs/Web/Guide/Using_FormData_Objects#Sending_files_using_a_FormData_object
-		if (analyze.form.attr('method').toLowerCase() === 'post' && 'FormData' in window) {
+		if (analyze.form.attr('method') && analyze.form.attr('method').toLowerCase() === 'post' && 'FormData' in window) {
```
Comment (fprochazka) [inline nette.ajax.js]: i think that nicer would be to save it into a variable

Comment (Ciki) [inline nette.ajax.js]: or maybe not use `attr()` at all & use `form.method` directly .. ?

**Comment by Ciki** [comment]
@vojtech-dobes @fprochazka should be fine now.. can you merge?

**Comment by vojtech-dobes** [comment]
Can you please squash it to 1 commit?

**Comment by Ciki** [comment]
can I do it online on GH? I'm not on pc now..

**Comment by vojtech-dobes** [comment]
Unfortunately no. When you will squash it, you can keep message from the first commit. Ideally prefix `Forms: ` would be great, so result would be:

```
Forms: fix calling toLowerCase() on undefined
```

**Comment by Ciki** [comment]
I recreated it in #120

**Pull request closed without merging**